app/javascript/components/products/cardProduct.scss

```diff
@@ -0,0 +1,103 @@
+@import '../.././styles/main.scss';
+
+%text-properties {
+  font-size: 14px;
+  font-weight: 900;
+  line-height: 1.25;
+}
+
+.card-product {
+  border-radius: 8px;
+  border: 1px solid $black;
+  box-shadow: 3px 3px 6px 0 rgba(1, 1, 1, 0.29);
+  height: 189px;
+  margin-bottom: 20px;
+  width: 153px;
+
+  @media screen and (min-width: 700px) {
```

Comment:
check if we already have a variable for this resolution

Reply:
Done